onnxruntime/test/shared_lib/test_inference.cc

```diff
@@ -100,7 +107,6 @@ void TestInference(Ort::Env& env, T model_uri,
   }
 
   if (custom_op_library_filename) {
-    void* library_handle = nullptr;  // leak this, no harm.
     Ort::ThrowOnError(Ort::GetApi().RegisterCustomOpsLibrary((OrtSessionOptions*)session_options, custom_op_library_filename, &library_handle));
```

Comment:
> &l [](start = 125, length = 3)

The function above takes this as a reference to poiner. Essentially a pointer to pointer. Now you are taking an address of a reference. This is a little confusing

Reply:
The function takes a pointer to a pointer is it not ? We get a reference to a pointer above.

Reply:
Simplified everything - should be much cleaner now